azure/services/managedclusters/managedclusters.go

```diff
@@ -22,18 +22,22 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerservice/armcontainerservice/v4"
 	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/utils/ptr"
 	infrav1 "sigs.k8s.io/cluster-api-provider-azure/api/v1beta1"
 	"sigs.k8s.io/cluster-api-provider-azure/azure"
 	"sigs.k8s.io/cluster-api-provider-azure/azure/services/async"
+	"sigs.k8s.io/cluster-api-provider-azure/azure/services/token"
 	"sigs.k8s.io/cluster-api-provider-azure/util/reconciler"
 	"sigs.k8s.io/cluster-api-provider-azure/util/tele"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
-const serviceName = "managedcluster"
-
-const kubeletIdentityKey = "kubeletidentity"
+const (
+	serviceName        = "managedcluster"
+	kubeletIdentityKey = "kubeletidentity"
+	aadResourceID      = "6dae42f8-4368-4678-94ff-3960e28e3630"
```

Comment:
Just for my own understanding: is this supposed to be hard-coded?

Reply:
yes

https://azure.github.io/kubelogin/concepts/aks.html?highlight=6dae42f8-4368-4678-94ff-3960e28e3630#azure-kubernetes-service-aad-server

Reply:
Can you please add a comment with the link for future reference?

Reply:
ack